people/bjorn3.toml

```
@@ -0,0 +1,4 @@
+name = "bjorn3"
+github = "bjorn3"
+github-id = 17426603
+zulip-id = 133247
```

```suggestion
zulip-id = 133247
email = "bjorn3_gh@protonmail.com"
```
made a new email address for this